src/main/resources/data/ae2/tags/blocks/blacklisted/spatial.json

```diff
@@ -1,7 +1,6 @@
 {
   "replace": false,
   "values": [
-    "#spectrum:budding_blocks",
```

Comment:
This entry here mainly is:
- to give players a reason to go back to the DD from time to time
- make them not complain when they notice that moved budding blocks can not be used to generate ink, once the ink update it out (which will still take a while and will prob. be 1.21+, so eh)

Reply:
doesn't the resonance enchantment have these same issues anyways?